test/test_cliptok.py

```diff
@@ -105,6 +106,11 @@ def test_tokenizer(self):
         self._run_tokenizer(["Testing multiple      sequences       of spaces"])
         self._run_tokenizer(["      in the beginning and the end.      "])
         self._run_tokenizer([" "])
+        
+        # Currently, HF CLIPTokenizer and CLIPTokenizerFast implementations are not consistent with apostrophe handling
+        # until ftfy is installed so we test aposrophe handling for both slow and fast Python implementations.
+        self._run_tokenizer(["Testing words with apostrophes such as you're, i'm, don't, etc."], use_slow=True)
```

Comment:
generally, we don't test other components buggy behavior, just make sure ours align with the expected behavior.

Reply:
done!